docs/ios-setup.md

```diff
@@ -0,0 +1,98 @@
+# Installation
+
+## With CocoaPods
+
+If you don't have a Podfile in the `ios` folder of your React Native project, follow this guide to create one: https://facebook.github.io/react-native/docs/integration-with-existing-apps#configuring-cocoapods-dependencies
+
+Add the following to your Podfile:
+
+  ```ruby
+  pod "react-native-heap", path: "../node_modules/@heap/react-native-heap"
+  ```
+
+Then run:
+
+  ```bash
+  pod install
+  ```
+
+You're done! :tada:
+
+## Without Cocoapods
+
+All terminal commands assume you are in the top-level directory of your React Native project.
+
+1. Locate the Xcode project for RNHeap: `open node_modules/@heap/react-native-heap/ios` . You should see a Finder window containing the `RNHeap.xcodeproj` file.
+1. Open the Xcode project for your native app, in the `ios` directory of your project.
+1. Drag the `RNHeap.xcodeproj` file from the Finder window into the "Libraries" group of your Xcode project.
+1. Select your project name in the Xcode project navigator. Select the "Build Phases" tab for your app's target.
+1. Expand the build phase called "Link Binary With Libraries". In the project navigator on the left, expand the `RNHeap.xcodeproj` entry, and find `libRNHeap.a` in the "Products" group. The icon for `libRNHeap.a` should look like a little building with columns.
+1. Drag the `libRNHeap.a` file into the list of other libraries to be linked.
+1. Expand the build phase called "Copy Bundle Resources". In the project navigator on the left, expand the `RNHeap.xcodeproj` entry, and find `HeapSettings.bundle`. Drag this into the list of bundle resources to be copied.
+
+**NOTE**: Using `react-native link` is not currently supported
+
+# Initialization
+
+- This library needs to be initialized so events are sent to the right application/environment ID. 
+Place a `heap.config.json` file at the root of your application's repository with this structure, substituting your own application/environment IDs.
+
+  ```json
+  {
+    "default": {
+      "heapAutoInit": true
+    },
+    "dev": {
+      "heapAutoInit": true,
+      "heapAppId": "11"
+    },
+    "prod": {
+      "heapAppId": "12",
+      "heapAutoInit": true
+    }
+  }
+  ```
+
+- Note that different application/environment IDs can be set for development and production. Heap can also be enabled/disabled on a per-environment basis. Values in `default` are used if a key is missing in either `dev` or `prod`.
+
+## Manual Initialization
+
+- If you'd like to skip auto-initialization, simply call `Heap.setAppId` with an application/environment ID.
```

Comment:
I don't think users know what "auto-initialization" means. Why don't we say:
> If you'd like fine-grained control over when the Heap library initializes, call `Heap.setAppId` with an application/environment ID. (Most users won't need this.)

Reply:
That's a lot better, thanks! 👍